<?xml version="1.0" encoding="utf-8"?>
<report
	progress="">
	<activity
		result="Failed"
		duration="1.4m"
		durationms="82665"
		type="root"
		rid="9481042800aeea8"
		totalerrorcount="1"
		totalwarningcount="0"
		totalsuccesstestcasecount="0"
		totalfailedtestcasecount="0"
		totalblockedtestcasecount="0"
		childsuccesscount="0"
		childfailedcount="0"
		childblockedcount="0"
		totalsuccesscount="0"
		totalfailedcount="0"
		totalblockedcount="0"
		totalmaintenancemodecount="0"
		user="Test"
		rxversion="9.1.0.0"
		host="THSCVM10050B"
		osversion="Windows 7 Service Pack 1 64bit"
		runtimeversion="4.0.30319.42000"
		procarch="32bit"
		oslanguage="en-US"
		dtnlanguage="en-US"
		screenresolution="1364x768"
		timestamp="6/10/2019 3:33:34 PM"
		timestampiso="2019-06-10T10:03:34"
		endtime="6/10/2019 3:34:56 PM"
		timeoutfactor="1"
		module="ToCraeteNewEstimate">
		<detail>
			ToCraeteNewEstimate
		</detail>
		<varbindings>
		</varbindings>
		<item
			timeRelativeToTestSuiteStartTime="00:00.000"
			timeRelativeToTestModuleStartTime="01:22.666"
			timeWallClock="15:34:56"
			time="00:00.000"
			level="Error"
			category="Module">
			<message>
				The test run has been aborted by the user.
			</message>
			<metainfo
				loglvl="Error">
			</metainfo>
		</item>
	</activity>
</report>
